### ISSUE TITLE
How to draw Expected Average Overlap (EAO) scores for each attribute

### ISSUE THREAD

**Issue by jananithangavel** [votchallenge/toolkit#62]

**Comment by davidyang180** [comment]
Yes! I also want to know how to use the VOT toolkit to get the EAO scores of each attribute subsequence, such as illumination changes, occlusion attribute subsequences, I sincerely hope the author can help answer. @lukacu
